tap_twitter_ads/streams.py

```diff
@@ -292,6 +295,9 @@ def sync_endpoint(self,
             # API Call
             cursor = self.get_resource(stream_name, client, path, new_params)
 
+            # Cursor for children to retrieve parent_ids
+            cursor_child = copy.deepcopy(cursor)
```

Comment:
Don't iterate records twice, In case of parent and child both are selected, use a minimum of parent's bookmark and child's bookmark.